doc/safety/safety_overview.rst

```diff
@@ -0,0 +1,263 @@
+**repercussions** on other components.
+
+Encapsulated single functionality (if not reasonable and manageable in safety)
+------------------------------------------------------------------------------
+
+Another requirement for the overall system and software environment is that individual
+functionalities can be disabled within components. This is because if a function is absolutely
+unacceptable for safety (e.g. complete dynamic memory management), then these individual
+functionalities should be able to be turned off. The Zephyr Project already offers such a
+possibility through the use of Kconfig and its flexible configurability.
+
+Processes and workflow
+**********************
+
+.. figure:: images/zephyr-safety-process.svg
+   :align: center
+   :alt: Safety process and workflow overview
+   :figclass: align-center
+
+   Safety process and workflow overview
+
+The diagram describes the rough process defined by the Safety Committee to ensure safety in the
+development of the Zephyr project. To ensure understanding, a few points need to be highlighted and
+some details explained regarding the role of the safety architect and the role of the safety
+committee in the whole process. The diagram only describes the paths that are possible when a
+change is related to safety.
+
+#. On the main branch, the safety scope of the project should be marked, which typically represents
+   a small subset of the entire code base. This marked area should then be made auditable during
+   normal development on “main”, which means that special attention is paid to quality goals
+   (`Quality`_) and safety processes within this scope. The Safety Architect works alongside the
+   general software architect in this area, monitoring the development process to ensure that the
+   architecture meets the safety requirements.
+
+#. At this stage, the safety architect plays an increasingly important role. For PRs/issues that
+   fall within the safety scope, the safety architect should ideally be involved in the discussions
+   and decisions of minor changes in the safety scope to be able to react to safety-relevant
+   changes that are not conformant. If a pull request or issue introduces a significant and
+   influential change or improvement that requires extended discussion or decision-making, the
+   safety architect should bring it to the attention of the Safety Committee or the Technical
+   Steering Committee (TSC) as appropriate, so that they can make a decision on the best course of
+   action.
+
+#. This section describes the certification side. At this point, the code base has to be in an
+   "auditable" state, and ideally no further changes should be necessary or made to the code base.
+   There is still a path from the main branch to this area. This is needed in case a serious bug or
+   important change is found or implemented on the main branch after the auditable branch was
```

Comment:
main branch or LTS? Auditable is besed on LTS, so it needs to go into LTS first.

Reply:
Right, reworded the paragraph a little bit to clarify the situation and also adapted the diagram.